puffin_egui/Cargo.toml

```
@@ -22,7 +22,7 @@ include = [
 
 [dependencies]
 chrono = "0.4"
-egui = "0.19.0"
+egui = { version = "0.20.0", default-features = false, features = ["default_fonts"] }
```

```suggestion
egui = { version = "0.20.1", default-features = false, features = ["default_fonts"] }
```

I made a patch-release yesterday fixing key-repeates